### ISSUE TITLE
Currently not working

### ISSUE THREAD

**Issue by JakeH** [reustle/covid19japan#63]
Loading the page at https://covid19japan.com/ results in no data displayed and the following error in console. 

```
main.js:81 Uncaught (in promise) TypeError: Cannot read property 'length' of undefined
    at pullLatestSumAndDiff (main.js:81)
    at calculateTotals (main.js:90)
    at (index):225
    at main.js:56
```

**Comment by olingern** [comment]
@reustle It looks like the JSON is being written incorrectly because the google sheet indexes are off. 

```javascript
const SHEET_DAILY_SUM_TAB = 3;
const SHEET_LAST_UPDATED_TAB = 4;
```

These two seem to be off by one. Changing them to 4 and 5 respectively fixes it locally for me. Was **Patient Statuses** always in the google sheet?

**Comment by reustle** [comment]
Sorry, never got tagged in this so I didn't get an email :( 

Trying to fix now

**Comment by olingern** [comment]
Looks like the **Patient Status** was moved further down the list. Looks like the action ran again and wrote the correct JSON

**Comment by reustle** [comment]
Fixed! Really need to get some data checking automated :S

The sheet order was completely messed up, not sure if some sheet bug or human error, but its solved. Would love to get off Google Sheets soon 🙌

**Comment by olingern** [comment]
@reustle what would that look like? I'd be interested in putting something together for node + postgres or mongo ([mLab](https://mlab.com/plans/pricing/#plan-type=sandbox) being one of the few database services with a free tier)

**Comment by reustle** [comment]
MVP could be to modify the existing node script to pull all sheets (index 1-10 or something) and guess each sheet based on the keys found, and extracts the data it needs to generate the current JSON output file. Thoughts?

**Comment by olingern** [comment]
> Would love to get off Google Sheets soon raised_hands

I think that could get things a bit more stable and fix #42, but doesn't get things off of google sheets. I think getting off of google sheets could be a fairly large task due to needing input from users and approval from admins.

I guess ideally, you'd have a database with an admin area where admin can approve user reports and the task would look at that database rather than google sheets. Seems like overkill given how simple things are right now, but would be worthwhile if the virus sticks around for a while.

**Comment by reustle** [comment]
I agree, setting up a simple Django instance was a plan for some time but was actually more worried about how to nicely do the UI that is collaborative that lets many editors not overwrite each others changes (google sheets does this well).
